flows/events/run_result_changed.go

```diff
@@ -32,16 +32,17 @@ type RunResultChangedEvent struct {
 	BaseEvent
 	callerOrEngineEvent
 
-	Name              string         `json:"name" validate:"required"`
-	Value             string         `json:"value"`
-	Category          string         `json:"category"`
-	CategoryLocalized string         `json:"category_localized,omitempty"`
-	NodeUUID          flows.NodeUUID `json:"node_uuid" validate:"required,uuid4"`
-	Input             *string        `json:"input,omitempty"`
+	Name              string            `json:"name" validate:"required"`
+	Value             string            `json:"value"`
+	Category          string            `json:"category"`
+	CategoryLocalized string            `json:"category_localized,omitempty"`
+	NodeUUID          flows.NodeUUID    `json:"node_uuid" validate:"required,uuid4"`
+	Input             *string           `json:"input,omitempty"`
+	Extra             map[string]string `json:"extra,omitempty"`
```

Comment:
For now this just occurs on the event and doesn't go into the actual result dict